airbyte-integrations/connectors/source-coingecko-coins/acceptance-test-config.yml

```diff
@@ -1,20 +1,28 @@
 # See [Source Acceptance Tests](https://docs.airbyte.com/connector-development/testing-connectors/source-acceptance-tests-reference)
 # for more information about how to configure these tests
 connector_image: airbyte/source-coingecko-coins:dev
-tests:
+acceptance_tests:
   spec:
-    - spec_path: "source_coingecko_coins/spec.yaml"
+    tests:
+      - spec_path: "source_coingecko_coins/spec.yaml"
+        backward_compatibility_tests_config:
+          disable_for_version: "0.1.0"
   connection:
-    - config_path: "secrets/config.json"
-      status: "succeed"
-    - config_path: "integration_tests/invalid_config.json"
-      status: "failed"
+    tests:
+      - config_path: "secrets/config.json"
+        status: "succeed"
+        timeout_seconds: 120
+      - config_path: "integration_tests/invalid_config.json"
+        status: "failed"
   discovery:
-    - config_path: "secrets/config.json"
+    tests:
+      - config_path: "secrets/config.json"
   basic_read:
-    - config_path: "secrets/config.json"
-      configured_catalog_path: "integration_tests/configured_catalog.json"
-      empty_streams: []
+    tests:
+      - config_path: "secrets/config.json"
+        configured_catalog_path: "integration_tests/configured_catalog.json"
+        empty_streams: []
   full_refresh:
```

Comment:
Looks like full refresh test is still failing

Reply:
Yes, i wrote the comment under the test, it's because of run on active account and data is bot frozen.